### ISSUE TITLE
Adding the plugin and not compiling [Android]

### ISSUE THREAD

**Issue by otavio515** [NiklasMerz/cordova-plugin-fingerprint-aio#97]
Hello, I'd like to ask if you could improve the documentation with minimun android SDK or Version once that I'm having trouble when I create a new project and add this plugin apparently. If I just create a blank app and do:

```
npm install
ionic cordova platform add android
ionic cordova build android
ionic cordova run android

```
there is no problem, it works perfectly, but if I add the plugin, everything falls.

I'll make an edit later because I'm just restarting another project exactly to show the errors which I'm talking about after adding the plugin.

I'm not sure if cordova update to 8.0.0 is the main problem, I'm using Xiaomi MI 5 on Android 7.0.0

I appreciate since now.

**Comment by NiklasMerz** [comment]
Thanks for reporting. I think the minim SDK is 23 and I added this to README.

This problem should be fixed with #94 . Please try installing the release candidate:

```
cordova plugin add cordova-plugin-fingerprint-aio@rc
```

I will publish this as the version later.

**Comment by NiklasMerz** [comment]
This should be resolved with version 1.3.6
